src/SqlProviderBase.ts

```diff
@@ -196,23 +195,20 @@ export abstract class SqlProviderBase extends NoSqlProvider.DbProvider {
                                 }
                             });
                         });
-                        dropQueries = _.flatten(_.chain(tableNames)
-                            .filter(name => !_.includes(tableNamesNeeded, name))
+                        dropQueries = _.flatten(
+                            _.filter(tableNames, name => !_.includes(tableNamesNeeded, name))
                             .map(name => {
```

Comment:
You want this to be _.map now too.  Probably just break it out into separate lines to make it clearer rather than triple-nest at this point.